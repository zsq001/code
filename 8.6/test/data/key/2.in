100010111010001011101000101110100010111010001011101000101110100010111010001011101000
