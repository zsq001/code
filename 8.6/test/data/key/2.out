16
8 14 16 23 29 31 38 44 46 53 59 61 68 74 76 83
